src/Platform.c

```diff
@@ -162,6 +202,12 @@ int PltCreateEvent(PLT_EVENT* event) {
     }
 
     return 0;
+#elif defined(__vita__)
+    event->mutex = sceKernelCreateMutex("", 0, 0, NULL);
+    event->cond = sceKernelCreateCond("", 0, event->mutex, NULL);
+    event->signalled = 0;
+    printf("mutex: 0x%x cond: 0x%x\n", event->mutex, event->cond);
```

Comment:
It also looks like you should check if sceKernelCreateMutex or sceKernelCreateCond returned null.